text/0010-gatsby-docs-localization.md

```diff
@@ -0,0 +1,210 @@
+* Start Date: 2019-09-11
+* RFC PR:
+* Gatsby Issue:
+
+
+## Summary
+
+Begin the process of localizing the documentation of Gatsby into non-English languages.
+
+
+## Motivation
+
+[You Belong Here.](https://www.gatsbyjs.org/blog/2018-09-07-gatsby-values/#you-belong-here)
+
+Gatsby believes that everyone belongs in Open Source, and that includes developers that do not speak English. Having access to translated documentation opens up the ability to create websites to people who don't speak English and don't have the privileges to be able to learn.
+
+
+## Detailed Design
+
+The translations will be done in the [same monorepo](https://github.com/gatsbyjs/gatsby) as the rest of the Gatsby documentation and code. Additional tooling, automation, and documentation will be made to ensure that the languages stay up-to-date and to maintain translation quality.
+
+### Overview of Changes
+
+The following things need to be created:
+
+* [ ] A bot to track changes to the English docs and create pull requests
```

Comment:
Since gatsbyjs is a monorepo (Compared to reactjs as many repos):

That means that in case of an commit (even changing a typo) then the bot generates a pull request per Language - if we have in react about 30 languages, then we get a wave of 30 pull requests to the monorepo?)

Reply:
Even in React, which has 45 different translations in progress, we only have around 15 with "complete" translations, and a handful more with maybe 50% completion. The bot will only make pull requests to languages that have translations: for most pages, this will be the 15 completed translations plus one or two more.

Also, even though right now it says "every commit", the React bot runs on a weekly schedule: it batches all the updates from within a week. So we can do something similar or even run on different schedules per language: more popular languages can run the bot with every commit, while less active languages can run once a week, or even not at all.

Reply:
Thx for info

How about to make for each language one repo (like React) and then push/sync completed pages to the monorepo?

Then each language can have it own issues and tracking...